### DISCUSSION TITLE
Using wiki.js in Microsoft Teams desktop app's Tab

### DISCUSSION THREAD

**Discussion by Mukhammadjon-Jalolov** [requarks/wiki#7168] [Help / Questions]
Hello everyone!
I have a wiki.js running on Azure with Azure AD authentication. And now I have uploaded a manifest.json file to Microsoft Teams Tab and it is showing my wiki.js app well. So basically I need a wiki.js showing on my Teams tab with authentication.
However, any of the Authentication is NOT working (both local and Azure AD). **It works fine on a browser.** What could be reason for that and how that can be fixed? I appreciate any response/solution to this case. Thank you.
![teams tab](https://github.com/requarks/wiki/assets/38403832/d9f48bda-f376-41da-a408-92479e87acc9)

**Comment by NGPixel** [comment]
Stop creating duplicate posts.
